### PR TITLE
Added repo badges in the readme

### LINKED ISSUE #48: 📚 [Documentation] - Adding repo badges in README.
We can add badges in the ReadME that show the statistics of the repository.

Something like this maybe?
2023-05-30 10_49_15-apoorvaron_Shorty_ • Shortens link and customize it according to user’s need  -

@praveenscience , I'd like to work on this issue.

### PR DESCRIPTION
**Pull request by mukul314**
## Related Issue(s)

Closes #48 

## Proposed Changes

Added repo badges in readme

## Type of Change

- [ ] New Feature (Landing Page)
- [ ] New Feature (App)
- [ ] Bug (Landing Page)
- [ ] Bug (App)
- [ ] New Component (App)
- [ ] Enhancement
- [x] Documentation / ReadMe / Workflows
- Other (If not in above options):  

## Your Idea for Level (GSSoC)

- [x] Level 1
- [ ] Level 2
- [ ] Level 3

### CODE REVIEW

**Comment by AmanJain18** [comment]
@mukul314  Edit this PR template and rewrite it properly

**Comment by mukul314** [comment]
> @mukul314 Edit this PR template and rewrite it properly

@AmanJain18 done!

**Comment by AmanJain18** [comment]
Also fix the title bro.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+<img src="https://img.shields.io/github/issues-closed-raw/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+<img src="https://img.shields.io/github/issues/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+<img src="https://img.shields.io/github/issues-closed/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+<img src="https://img.shields.io/github/stars/CatsInTech/GitHub-ReadMe.svg?style=social&label=Stars" />
+<img src="https://img.shields.io/github/forks/CatsInTech/GitHub-ReadMe.svg?style=social" />
+<img src="https://img.shields.io/github/issues-pr/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+<img src="https://img.shields.io/github/last-commit/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+<img src="https://img.shields.io/github/contributors/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+<img src="https://img.shields.io/github/issues-pr-closed-raw/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+</div>
```
Comment (praveenscience) [inline README.md:17]: Can we have this aligned properly, thanks! 😊

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
-<img src="https://img.shields.io/github/issues/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
-<img src="https://img.shields.io/github/issues-closed/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
-<img src="https://img.shields.io/github/stars/CatsInTech/GitHub-ReadMe.svg?style=social&label=Stars" />
-<img src="https://img.shields.io/github/forks/CatsInTech/GitHub-ReadMe.svg?style=social" />
-<img src="https://img.shields.io/github/issues-pr/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
-<img src="https://img.shields.io/github/last-commit/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
-<img src="https://img.shields.io/github/contributors/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
-<img src="https://img.shields.io/github/issues-pr-closed-raw/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+<div style="text-align: center;">
+  <p>
```
Comment (praveenscience) [inline README.md:2]: Do we require the `<p>` tag?

File: README.md [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
+    <img src="https://img.shields.io/github/issues-closed-raw/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+    <img src="https://img.shields.io/github/issues/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+    <img src="https://img.shields.io/github/issues-closed/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+    <img src="https://img.shields.io/github/stars/CatsInTech/GitHub-ReadMe.svg?style=social&label=Stars" />
+    <img src="https://img.shields.io/github/forks/CatsInTech/GitHub-ReadMe.svg?style=social" />
+    <img src="https://img.shields.io/github/issues-pr/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+    <img src="https://img.shields.io/github/last-commit/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+    <img src="https://img.shields.io/github/contributors/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+    <img src="https://img.shields.io/github/issues-pr-closed-raw/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+  </p>
```
Comment (praveenscience) [inline README.md:18]: Also, can you move it below the ToC?

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -27,6 +27,28 @@
...
+    <img src="https://img.shields.io/github/issues-closed/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+    <img src="https://img.shields.io/github/stars/CatsInTech/GitHub-ReadMe.svg?style=social&label=Stars" />
+    <img src="https://img.shields.io/github/forks/CatsInTech/GitHub-ReadMe.svg?style=social" />
+    <img src="https://img.shields.io/github/issues-pr/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+    <img src="https://img.shields.io/github/last-commit/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+    <img src="https://img.shields.io/github/contributors/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+    <img src="https://img.shields.io/github/issues-pr-closed-raw/CatsInTech/GitHub-ReadMe.svg?style=for-the-badge" />
+
+</div>
+<br>
```
Comment (praveenscience) [inline README.md:49]: Amazing, please kindly indent. 😊 I'll merge it ASAP. Use `<br />` as well instead of `<br>`. Thanks.

**Review by praveenscience — CHANGES_REQUESTED** [review]

**Comment by praveenscience** [comment]
@AmanJain18 It's alright, @mukul314 is trialling a lot of things and then again, getting merge conflicts and stuff! 😅

**Comment by AmanJain18** [comment]
Ya No Issues. 😊🤗

**Comment by mukul314** [comment]
> For just adding badges in readme, this much Commits? 😂🤦

@AmanJain18 🤣 , I know that this issue has been stretched a bit, but I learned a lot from it also. Along with that, I would also like to thank amazing maintainers like @praveenscience and you who have patiently reviewed each and every request and responded timely. The guidance that I received is remarkable. Thanks a lot for that 😄 .

As for the changes, I will replace the <br> with </br> but can you tell me how to keep that indentation part? I guess I am not getting what you want to convey 😅 .

File: README.md (hunk 4 above)
Comment (praveenscience) [reply README.md]: Never mind, I did it myself. 😊

**Review by praveenscience — APPROVED** [review]
Never mind, @mukul314, I have made it myself, check it out from the changes! 💪

**Pull request merged**